test_sets/Defects/7_0/3174.txt

```diff
@@ -0,0 +1 @@
+./test_scripts/Defects/7_0/3174_App_unregistering_Force_close_Nav_app.lua
```

Comment:
Instead of creating a new set please add script into https://github.com/smartdevicelink/sdl_atf_test_scripts/blob/develop/test_sets/Defects/Defects_release_7_0.txt

Reply:
Done